### ISSUE TITLE
CSS | Focus outline not appearing

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/vets-design-system-documentation#5153]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

While shifting focus to the error message inside of web components directly on the `role="alert"` span (see https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4993). The focus ring doesn't appear around the focused message:

<img width="754" height="722" alt="error message inside a va-radio has focus, but no focus ring appears" src="https://github.com/user-attachments/assets/0909e217-f71f-4d0c-87ef-b08f86b0c59b" />

This happens in multiple components, but I didn't have time to check all web components:

| Component | Shows outline on error? |
|----|:----:|
| va-checkbox-group | ❌ |
| va-radio | ❌ |
| va-select | ❌ |
| va-text-input | ❌ |
| va-memorable-date | ✅ |

## What I expected to happen

Focus outline should appear around the error message. While attempting to troubleshoot the problem, I noticed that the default outline definition is using `auto` instead of `solid` for the outline style - it appears that using `auto` will override the outline width, but it does show the outline:

<img width="760" height="732" alt="outline showing around the radio error message. The dev tools styling section shows that an 'auto' outline style is being applied" src="https://github.com/user-attachments/assets/30426255-e049-4e79-a4a6-1b109a7d6603" />

## Reproducing

- CSS Library version: Current
- Device: (e.g. iPhone 8, Macbook): Current
- Browser: (e.g. Firefox, IE 11): Current

**URL(s) where this bug occurs**

- https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start

**Steps to reproduce:**

1. Go to [Supplemental Claims subtask page](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start
2. Open the dev tools and inspect the va-radio label
3. In the elements panel, find and select the span with ID = `radio-error-message`
4. Click continue without making a choice 
5. You may need to click on the continue twice
6. Once the `radio-error-message` span has a `tab-index="-1"` you can examine the CSS changes to see that the outline is applied, but still isn't visible

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other
